nodebuilder/tests/reconstruct_test.go

```diff
@@ -103,9 +103,19 @@ func TestFullReconstructFromLights(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), swamp.DefaultTestTimeout)
 
 	t.Cleanup(cancel)
-	sw := swamp.NewSwamp(t, swamp.WithBlockTime(btime))
-	fillDn := sw.FillBlocks(ctx, bsize, blocks)
+	sw := swamp.NewSwamp(t, swamp.WithBlockTime(btime), swamp.WithLogLevel("ERROR",
+		"bitswap_network",
+		"bitswap-server",
+		"bitswap-client",
+		"net/identify",
+		"mocknet",
+		"pubsub",
+		"basichost",
+		"bs:sprmgr",
+		"dagstore/upgrader",
+	))
```

Comment:
Don't think it has to be an option as it applies the levels globally rather than to the specific swamp. 
When we do `make test` all the tests are run in one process with shared global space, meaning that once this test is executed it changes the logging for all subsequent tests and swamps, rather than the particular Swamp. 

Not a strong opinion here, it just doesn't make sense to me

Reply:
It makes a lot of sense when you are checking 9mb of logs instead of 500mb

Reply:
I am not saying to remove changing log level lol, but to make it directly with log.SetLogLevel rather than with the option

Reply:
It for different test you may need different logs. And I assume it will turn into a mess with manual log level editing. But I've got your point. Will try to find another pragmatic way.